src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -215,7 +215,7 @@ export const OpportunityCard = ({
       <div className="rounded shadow-border bg-white pt-5 pr-5 pl-5 sm:pl-6 pb-5 sm:pb-7">
         <div className="flex flex-col sm:flex-row sm:gap-5">
           <div className="flex flex-row justify-between align-center">
-            <div>
+            <div className={type != 'work' ? 'bg-todo' : undefined}>
```

Comment:
As a general rule In JS/TS, you should always favor triple operators instead of double, so `!==` instead of `!=`

Reply:
Fixed